R/impute_bag.R

```diff
@@ -180,6 +180,10 @@ step_impute_bag_new <-
 bag_wrap <- function(vars, dat, opt, seed_val) {
   seed_val <- seed_val[1]
   dat <- as.data.frame(dat[, c(vars$y, vars$x)])
+  if (is.character(dat[[vars$y]])) {
+    dat[[vars$y]] <- factor(dat[[vars$y]])
+  }
```

Comment:
Is this because `ipred::ipredbagg()` doesn't have a character method? So you've been getting this message?

```r
> ipred:::ipredbagg.default
function (y, ...) 
{
    stop(paste("Do not know how to handle objects of class", 
        class(y)))
}
```

Reply:
Yes.